txscript/data/script_tests.json

```diff
@@ -785,6 +788,7 @@
 ["22 21 20", "0 PICK 20 EQUALVERIFY 2DROP DROP 1", "CLEANSTACK", "OK"],
 ["22 21 20", "1 PICK 21 EQUALVERIFY 2DROP DROP 1", "CLEANSTACK", "OK"],
 ["22 21 20", "2 PICK 22 EQUALVERIFY 2DROP DROP 1", "CLEANSTACK", "OK"],
+["1 0x02 0x0000", "PICK DROP", "NONE", "ERR_MINIMAL_DATA"],
```

Comment:
Since these were split up from the section comment that described what they're doing, please add individual comments accordingly.